### PR TITLE
Update the DOM-Parsing IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-03-02 19:44:58
Finished: 2018-03-02 19:57:17





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25157)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/348381146)

**Comment by foolip** [comment]
Seems like there is no existing test under https://github.com/w3c/web-platform-tests/tree/master/domparsing. Can you add one?

(Also indentation needs updating.)

File: interfaces/DOM-Parsing.idl [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      [CEReactions, TreatNullAs=EmptyString] attribute DOMString outerHTML;
+      [CEReactions] void insertAdjacentHTML(DOMString position, DOMString text);
+    };
+
+    partial interface Range {
+      [CEReactions, NewObject] DocumentFragment createContextualFragment(DOMString fragment);
+    };
+
+    partial interface Text {
+      attribute boolean serializeAsCDATA;
```
Comment (foolip) [inline interfaces/DOM-Parsing.idl]: I've added a test, and a failure pointed to something odd here. `serializeAsCDATA` isn't actually in the spec, it's commented out:
https://github.com/w3c/DOM-Parsing/blob/ba763b4d81a5a1fb5ec37deebf93659d853850e7/index.html#L608

@mdittmer any idea why that might be happening? Isn't a real HTML parser being used?

**Review by foolip — APPROVED** [review]
LGTM % serializeAsCDATA

File: interfaces/DOM-Parsing.idl (hunk 1 above)
Comment (mdittmer) [reply interfaces/DOM-Parsing.idl]: Nope.

**Comment by mdittmer** [comment]
@lukebjerring is this still relevant or abandoned?

**Comment by lukebjerring** [comment]
Abandoned.

**Comment by lukebjerring** [comment]
Un-abandoned.

File: interfaces/DOM-Parsing.idl [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  [CEReactions, TreatNullAs=EmptyString] attribute DOMString outerHTML;
+  [CEReactions] void insertAdjacentHTML(DOMString position, DOMString text);
+};
+
+partial interface Range {
+  [CEReactions, NewObject] DocumentFragment createContextualFragment(DOMString fragment);
+};
+
+partial interface Text {
+  attribute boolean serializeAsCDATA;
```
Comment (foolip) [inline interfaces/DOM-Parsing.idl]: This bit still needs to be removed. Manually perhaps? Or by switching to Reffy as data source?

**Review by foolip — APPROVED** [review]

**Pull request merged**